scripts/complete_liquidity_provision.js

```diff
@@ -4,6 +4,7 @@ const { sleep } = require("./utils/js_helpers")
 
 const Contract = require("@truffle/contract")
 const {
+  fetchTokenInfoFromExchange,
```

Comment:
I guess this was defined in a previous PR. Why not fetch token info from EVM?

Reply:
Maybe a better name would be `fetchTokenInfoByExchangeId`.

Reply:
What is `ExchangeId`?

Reply:
There are currently two functions fetching token information from the EVM: `fetchTokenInfoAtAddresses` and `fetchTokenInfoFromExchange`, the latter relying on the former during its execution. So "fromExchange" really just means that you get the addresses of the tokens from the exchange using the input token ids, addresses which are then used to fetch token info from the EVM. The output of the two functions has the same structure. If you have better naming proposals I'm happy to change the names (but this is a change that touches basically every file, so we might want to do that only when there are no other PRs standing).

Reply:
Nah its all good. Thanks for clearing that up.